.expo/README.md

```diff
@@ -0,0 +1,15 @@
+> Why do I have a folder named ".expo" in my project?
```

Comment:
great file!! love how you keep track of the questions you have as well as the answers! before opening a pr remember to remove any files that aren't necessary to the functionality of the app since what you're pushing is what will be pushed to the main branch.